pkg/querier/frontend/frontend.go

```diff
@@ -185,21 +186,15 @@ func (f *Frontend) handle(w http.ResponseWriter, r *http.Request) {
 			"time_taken", queryResponseTime.String(),
 		}
 
-		// Extract the ID from the header of the request
-		id, _, err := user.ExtractOrgIDFromHTTPRequest(r)
-		if err != nil {
-			logMessage = append(logMessage, "tenant_id", id)
-		}
-
 		// Ensure the form has been parsed so all the parameters are present
 		err = r.ParseForm()
 		if err != nil {
-			level.Warn(util.WithContext(r.Context(), f.log)).Log("unable to parse form for request")
+			level.Warn(util.WithContext(r.Context(), f.log)).Log("msg", "unable to parse form for request", "err", err)
```

Comment:
Super small thing, just to be consistent across the codebase.

```suggestion
			level.Warn(util.WithContext(r.Context(), f.log)).Log("msg", "unable to parse form for request", "error", err)
```